.github/workflows/android_deploy_staging.yml

```diff
@@ -49,10 +50,16 @@ jobs:
       - name: Build Android apk
         run: flutter build apk --flavor staging --debug --build-number $GITHUB_RUN_NUMBER
 
+      - name: Generate release notes
+        id: generate-release-notes
+        run: |
+          echo "RELEASE_NOTE_CONTENT=$((git log -1 --merges | grep "\[") && echo "" || echo $(git log -1 --merges --format=%B))" >> $GITHUB_OUTPUT
```

Comment:
@doannimble can we try to print/echo this `RELEASE_NOTE_CONTENT` var to verify that we're able to generate correct notes from the workflow log first?

I would like to check it first from here to verify whether the release note in Firebase [comes from the default fallback or your generated one](https://nimble-co.slack.com/archives/C01NKCFNE6R/p1696392743509209?thread_ts=1696386481.267319&cid=C01NKCFNE6R) 🙏 
![image](https://github.com/nimblehq/flutter-templates/assets/16315358/22f0f548-71ea-4518-bfe5-585a733eaa9e)